Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -35,6 +35,23 @@ protected override void OnViewInstantiated()
             viewInstance.InputField.onSelect.AddListener(OnInputSelected);
             viewInstance.InputField.onDeselect.AddListener(OnInputDeselected);
             viewInstance.CloseChatButton.onClick.AddListener(CloseChat);
+            viewInstance.LoopList.InitListView(0, OnGetItemByIndex);
+        }
+
+        private LoopListViewItem2 OnGetItemByIndex(LoopListView2 listView, int index)
+        {
+            if (index < 0 || index >= chatMessages.Count)
+                return null;
+
+            ChatMessage itemData = chatMessages[index];
+
+            LoopListViewItem2 item = listView.NewListViewItem(itemData.SentByOwnUser ? "ChatEntryOwn" : "ChatEntry");
```

Comment:
`listView` has `ItemPrefabDataList`, can you check if you can get a name from there instead of hardcoding?